js/ftx.js

```diff
@@ -2282,4 +2284,47 @@ module.exports = class ftx extends Exchange {
         const result = this.safeValue (response, 'result', {});
         return this.parseFundingRate (result, market);
     }
+
+    async fetchBorrowRates (params = {}) {
+        await this.loadMarkets ();
+        const response = await this.privateGetSpotMarginBorrowRates ();
+        const timestamp = this.milliseconds ();
+        const result = this.safeValue (response, 'result');
+        const rates = [];
+        for (let i = 0; i < result.length; i++) {
+            const rate = result[i];
+            rates.push ({
+                'currency': this.safeCurrencyCode (this.safeString (rate, 'coin')),
+                'rate': this.safeNumber (rate, 'previous'),
+                'increment': 'hourly',
```

Comment:
i would change this string to something that is more machine parseable, perhaps one hour in milliseconds

Reply:
Would it just be better to convert the rate to a daily amount for FTX? It looks like it might be the only one that's hourly.

If not, can you think of a better key than hourly?